test/e2e/create_test.go

```diff
@@ -165,11 +165,6 @@ func (tc *testContext) testMachineConfiguration(t *testing.T) {
 
 	machines, err := tc.waitForWindowsMachines(int(gc.numberOfMachineNodes), "Provisioned", false)
 	require.NoError(t, err, "error waiting for Windows Machines to be provisioned")
-	if tc.CloudProvider.GetType() == config.AzurePlatformType {
```

Comment:
Without having full context of both commits you linked to, its not possible to understand why this change is being made.
Can you write the commit message in a way that states what this change is doing on its own? Think about someone looking back in commit history trying to figure out why this change was made.

Reply:
Does https://github.com/openshift/windows-machine-config-operator/pull/2002#issue-2075240715 suffice?

Reply:
I'd like to see it within the commit message saying this is already being tested somewhere, and where

Reply:
@sebsoto  I updated the description of the PR with the supporting information you are requesting. PTAL https://github.com/openshift/windows-machine-config-operator/pull/2002#issue-2075240715.

The PR description is accessible from the commit history and will help with context in case we need to revisit this. I leaned towards not pushing an update to avoid triggering all CI jobs again considering this PR is blocking other PRs.